common/colour/colour.go

```diff
@@ -0,0 +1,100 @@
+package colour
+
+import (
+	"os"
+	"strings"
+
+	tty "github.com/mattn/go-isatty"
```

Comment:
Generally you want to avoid using import aliases in golang  unless there is a conflict (the idea is that it's more farmiliar to readers when packages are always imported via the same prefix) 

Go ignores the `go-` prefix on the import so 
```
import  "github.com/mattn/go-isatty"
...
isatty.IsTerminal(...)
```